src/dc_receive_imf.rs

```diff
@@ -83,6 +83,7 @@ pub async fn dc_receive_imf(
     let mut sent_timestamp = 0;
     let mut created_db_entries = Vec::new();
     let mut create_event_to_send = Some(CreateEvent::MsgsChanged);
+    let mut is_dc_message: MessengerMessage = Default::default();
```

Comment:
Now it is unnecessary, could you revert introduction of this variable?

Reply:
Shouldn't the CI (clippy) have told me, too? Anyway, of course, I forgot this.